templates/gcp/fips.html

```diff
@@ -4,354 +4,446 @@
-            <td class="u-align--right" data-heading="Version">1.8.1</td>
-            <td class="u-align--right" data-heading="CMVP Certificate"><a href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/3748">3748</a></td>
-          </tr>
-        </tbody>
-      </table>
-      <br>
-      <table class="p-table--mobile-card">
-        <thead>
-          <tr>
-            <th class="u-align--right" style="width: 9%;">Ubuntu LTS</th>
-            <th style="width: 30%;">FIPS certified component</th>
-            <th>Description</th>
-            <th class="u-align--right" style="width: 10%;">Version</th>
-            <th class="u-align--right" style="width: 15%;">CMVP Certificate</th>
-          </tr>
-        </thead>
-        <tbody>
-          <tr>
-            <td class="u-align--right" data-heading="Ubuntu LTS">20.04</td>
-            <td data-heading="FIPS certified component">Linux kernel (generic)</td>
-            <td data-heading="Description">The Linux kernel cryptographic library</td>
-            <td class="u-align--right" data-heading="Version">5.4.0-1021</td>
-            <td class="u-align--right" data-heading="CMVP Certificate"><a href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/4127">4127</a></td>
-          </tr>
-          <tr>
-            <td class="u-align--right" data-heading="Ubuntu LTS">20.04</td>
-            <td data-heading="FIPS certified component">OpenSSL, OpenSSH client, OpenSSH server</td>
-            <td data-heading="Description">General purpose cryptographic library that includes TLS implementation</td>
-            <td class="u-align--right" data-heading="Version">1:8.2p1</td>
-            <td class="u-align--right" data-heading="CMVP Certificate"><a href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/3966">3966</a></td>
-          </tr>
-          <tr>
-            <td class="u-align--right" class="u-align--right" data-heading="Ubuntu LTS">20.04</td>
-            <td data-heading="FIPS certified component">StrongSWAN</td>
-            <td data-heading="Description">IPSec based VPN solution library</td>
-            <td class="u-align--right" class="u-align--right" data-heading="Version">5.8.2</td>
-            <td class="u-align--right" class="u-align--right" data-heading="CMVP Certificate"><a href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/4046">4046</a></td>
-          </tr>
-        </tbody>
-      </table>
-      <br>
-      <p><a href="https://assets.ubuntu.com/v1/8ad870d4-Ubuntu-Pro-FIPS-for-Google-Cloud-compliance-DS-14-12.22.pdf">Read the Ubuntu Pro FIPS datasheet&nbsp;&rsaquo;</a></p>
     </div>
-  </div>
 
-  <div class="u-fixed-width">
-    <hr class="p-separator">
+    <table class="p-table--mobile-card">
+      <thead>
+        <tr>
+          <th class="u-align--right" style="width: 9%;">Ubuntu LTS</th>
```

Comment:
The items under "Ubuntu LTS" should be aligned to the left, similar for the table below
<img width="67" alt="Screenshot 2024-09-03 at 12 15 53 PM" src="https://github.com/user-attachments/assets/7e761146-08b5-4d27-8800-09d4327ad52f">